app/models/bounty.rb

```diff
@@ -65,12 +65,11 @@ def self.all
   scope :active, lambda { where(status: Status::ACTIVE) }
   scope :refunded, lambda { where(status: Status::REFUNDED) }
   scope :paid, lambda { where(status: Status::PAID) }
-  scope :not_refunded, lambda { where("status < :status AND status > :status", status: Status::REFUNDED) }
-
+  scope :not_refunded, lambda { where("status != :status", status: Status::REFUNDED) }
```

Comment:
What is the reasoning behind this change?

Reply:
I just reverted it, the previous change broke the query.

Reply:
The fix for this actually needs to be `status < :status OR status > :status`

Reply:
Have you tested this?